data/arabic/articles/c8j91j2ljppo.json

```diff
@@ -1282,317 +1300,317 @@
+            "caption": "عادت فرق الإنقاذ للعمل بعد يوم واحد من إعلان السلطات التركية وقف جهود البحث والإنقاذ",
+            "copyrightHolder": "EPA",
             "originCode": "cpsprodpb",
+            "allowSyndication": true,
             "type": "image"
           },
           "options": {
             "isBreakingNews": false,
             "isFactCheck": false
           },
           "prominence": "special",
-          "id": "urn:bbc:ares::asset:arabic/middleeast-63868817",
+          "id": "urn:bbc:ares::asset:arabic/middleeast-64711983",
           "type": "cps"
         },
         {
-          "name": "100 امرأة: قائمة بي بي سي لأكثر النساء إلهاما وتأثيرا في عام 2022",
-          "summary": "موسم بي بي سي لقائمة النساء الملهمات يحتفي بعقد من العمل من أجل التغيير في عام صاخب شهد وصول نساء إلى مراكز الصدارة.",
+          "headlines": {
+            "headline": "لم شمل رضيعة سورية يتّمها الزلزال مع عمتها"
+          },
+          "locators": {
+            "assetUri": "/arabic/middleeast-64703601",
+            "curieCpsUrn": "urn:bbc:cps:curie:asset:aa0d28a2-3d49-481a-a2a0-bee547995805",
+            "assetId": "64703601",
+            "cpsUrn": "urn:bbc:content:assetUri:arabic/middleeast-64703601",
+            "curie": "http://www.bbc.co.uk/asset/aa0d28a2-3d49-481a-a2a0-bee547995805"
+          },
+          "summary": "عمة وزوجها يتبنيان رضيعة وُلدت تحت أنقاض مبنى منهار في سوريا، بعد أن أصبحت العضوة الوحيدة الناجية من زلزال هائل من أسرتها.",
+          "timestamp": 1676973435000,
+          "language": "ar",
+          "passport": {
+            "campaigns": [
+              {
+                "campaignId": "5a988e3e39461b000e9dabfb",
+                "campaignName": "WS - Keep me on trend"
+              }
+            ],
+            "taggings": []
+          },
+          "cpsType": "STY",
           "indexImage": {
-            "id": "127901372",
+            "id": "128693159",
             "subType": "index",
-            "href": "http://c.files.bbci.co.uk/6AAE/production/_127901372_100_women_index_promo_976.png",
-            "path": "/cpsprodpb/6AAE/production/_127901372_100_women_index_promo_976.png",
+            "href": "http://c.files.bbci.co.uk/173A3/production/_128693159_2023-02-18t180233z_190590754_rc2tdz9ff212_rtrmadp_3_turkey-quake-syria-baby.jpg",
+            "path": "/cpsprodpb/173A3/production/_128693159_2023-02-18t180233z_190590754_rc2tdz9ff212_rtrmadp_3_turkey-quake-syria-baby.jpg",
             "height": 549,
             "width": 976,
-            "altText": "100 women",
-            "copyrightHolder": "BBC",
+            "altText": "الطفلة عفراء.",
+            "caption": "الطفلة عفراء أنقذت بعد ولادتها وظلت مرتبطة بأمها المتوفاة بالحبل السري.",
+            "copyrightHolder": "Reuters",
+            "originCode": "cpsprodpb",
+            "allowSyndication": true,
             "type": "image"
           },
-          "uri": "https://www.bbc.com/arabic/resources/idt-75af095e-21f7-41b0-9c5f-a96a5e0615c1",
```

Comment:
Heya, these Arabic responses are taken directly from FABL and we wanted to know if we're okay to update them since there are few changes.